docs/Install/Installation.md

```diff
@@ -19,18 +19,37 @@ Requirements
 curl -sS https://getcomposer.org/installer | sudo php -- --install-dir=/usr/bin --filename=composer
 cd /opt/
 sudo git clone https://github.com/librenms/librenmsv2 librenmsv2
-sudo chown -R librenms:librenms /opt/librenmsv2/ -R
 cd librenmsv2
+sudo mkdir logs
+sudo chown -R librenms:librenms /opt/librenmsv2/ -R
+```
+
+```bash
 cp .env.example .env
 ```
 
 Please now edit the .env file and set your options.
 
 ```bash
+replace Travis DB configuration with the one used by Librenms (have a look at the config.php file in /opt/librenms directory)
+
+DB_HOST=127.0.0.1
+DB_DATABASE=librenms
+DB_USERNAME=username
+DB_PASSWORD=password
+```
+
+```bash
 composer install
 php artisan key:generate
 ```
 
+Make sure webserver can write logs in storage directory
+
+```bash
+sudo chown -R www-data:www-data storage
```

Comment:
I would rather see this be:
chmod -R g+w storage logs
usermod -a -G librenms www-data

And up near the section where you do the other chown.